lib/src/test/kotlin/starknet/provider/response/JsonRpcResponseTest.kt

```diff
@@ -190,4 +190,132 @@ class JsonRpcResponseTest {
         assertEquals("Internal error", exception.message)
         assertEquals("[\"Invalid message selector\",\"0x1234\"]", exception.data)
     }
+
+    @Test
+    fun `rpc provider parses batch callContract`() {
+        val mockResponse = """
+           [
+              {
+                "jsonrpc": "2.0",
+                "result": [
+                  "0x1e2e22799540",
+                  "0x0"
+                ],
+                "id": "0"
+              },
+              {
+                "jsonrpc": "2.0",
+                "result": [
+                  "0x1e2e22799540",
+                  "0x0"
+                ],
+                "id": "1"
+              }
+            ]
+        """.trimIndent()
+
+        val httpServiceMock = mock<HttpService> {
+            on { send(any()) } doReturn HttpResponse(true, 200, mockResponse)
+        }
+        val provider = JsonRpcProvider("", httpServiceMock)
+
+        val call = Call(
+            DevnetClient.ethErc20ContractAddress,
+            Felt.fromHex("0x2e4263afad30923c891518314c3c95dbe830a16874e8abc5777a9a20b54c76e"),
+            listOf(Felt.fromHex("0x07f6331182b9bcbf9c1a5943e309c05399a935d170f7f07494cdf7a174cd7527")),
```

Comment:
We should use something more similar to the other tests where entrypoint is a string. No reason for mock test to have complicated values so it's not clear what it would do if it was a regular test.

Reply:
I've changed it to `Felt.ZERO`.